groupAnagrams.js

```diff
@@ -0,0 +1,44 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/group-anagrams/
+
+const groupAnagrams = (strs) => {
+  let strsArray = strs;
+  const result = [];
+  while (strsArray.length > 0) {
+    const grouppedAnagrams = [strsArray[0]];
+    for (let j = 1; j < strsArray.length; j += 1) {
+      if (strsArray[0].length !== strsArray[j].length) {
+        continue;
+      }
+      let letters = strsArray[j].split('');
+      let firstWordLetters = strsArray[0].split('');
```

Comment:
use `grouppedAnagrams ` instead of `strsArray[0]`

Reply:
Also, lets move this definition right next to `const grouppedAnagrams = [strsArray[0]];` (there is no need to initialize this inside of `for (let j = 1; j < strsArray.length; j += 1) {` cycle each iteration (also **pay attention to things like this** when doing self-review!).

Reply:
const grouppedAnagrams is different from strsArray[0], because I dynamically change strsArray.